docs/docs/Components/components-data.mdx

```diff
@@ -67,6 +67,10 @@ For provider-specific API components, see <Icon name="Blocks" aria-hidden="true"
 
 ### API Request parameters
 
+:::info SSRF Protection
+Starting in Langflow version 1.7, the **Follow Redirects** parameter is disabled (`false`) by default to prevent SSRF bypass attacks where a public URL redirects to internal resources. Only enable redirects if you trust the target server.  For more information, see [SSRF protection environment variables](/api-keys-and-authentication#ssrf-protection).
+:::
```

Comment:
Move this inside the param table in the follow_redirects row.

Also be sure to add a release note.